jdaviz/configs/default/plugins/export_plot/export_plot.vue

```diff
@@ -25,11 +26,86 @@
         <v-btn
          color="primary"
          @click="() => save_figure('svg')"
+         :disabled="movie_recording"
         >
          Export to SVG
         </v-btn>
        </v-list-item>
       </v-list>
+
+      <v-row v-if="config==='cubeviz'">
+        <v-expansion-panels accordion>
+          <v-expansion-panel>
+            <v-expansion-panel-header v-slot="{ open }">
+              <span style="padding: 6px">Export to Video</span>
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <v-row v-if="movie_msg!==''">
+                <span class="v-messages v-messages__message text--secondary" style="color: red !important">
+                    {{ movie_msg }}
+                </span>
+              </v-row>
+              <v-row v-if="movie_msg===''" class="row-no-outside-padding row-min-bottom-padding">
+                <v-col>
+                  <v-text-field
+                    v-model="i_start"
+                    class="mt-0 pt-0"
+                    type="number"
+                    label="Start"
+                    hint="Start Slice"
+                    persistent-hint
+                  ></v-text-field>
+                </v-col>
+                <v-col>
+                  <v-text-field
+                    v-model="i_end"
+                    class="mt-0 pt-0"
+                    type="number"
+                    label="End"
+                    hint="End Slice"
+                     persistent-hint
+                  ></v-text-field>
+                </v-col>
+              </v-row>
+              <v-row v-if="movie_msg===''" class="row-no-outside-padding row-min-bottom-padding">
+                <v-col>
+                  <v-text-field
+                    v-model="movie_fps"
+                    class="mt-0 pt-0"
+                    type="number"
+                    label="FPS"
+                    hint="Frame rate"
+                     persistent-hint
+                  ></v-text-field>
+                </v-col>
+              </v-row>
+              <v-row v-if="movie_msg===''" class="row-no-outside-padding row-min-bottom-padding">
+                <v-col>
+                  <v-text-field
+                    v-model="movie_filename"
+                    class="mt-0 pt-0"
+                    label="Filename"
+                    hint="Movie filename"
+                     persistent-hint
+                  ></v-text-field>
+                </v-col>
+              </v-row>
+              <v-row v-if="movie_msg===''" justify='end'>
+                <v-btn
+                 color="primary"
+                 @click="() => save_movie('mp4')"
+                 :disabled="movie_recording"
+                >
+                 Export to MP4
+                </v-btn>
+                <v-btn color="primary" icon @click="kill_recording" :disabled="!movie_recording">
+                  <v-icon>stop</v-icon>
+                </v-btn>
+              </v-row>

```

Comment:
both of these would benefit from tooltips.  Should the stop button be disabled or not visible at all before pressing the export button?

Reply:
The stop button is disabled when movie isn't recording (gray/unclickable vs blue/clickable). I guess I'll just hide it too as you suggested.